src/nl/hannahsten/texifyidea/index/LatexTodoSearcher.kt

```diff
@@ -0,0 +1,36 @@
+package nl.hannahsten.texifyidea.index
+
+import com.intellij.openapi.application.QueryExecutorBase
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.PsiFile
+import com.intellij.psi.search.IndexPattern
+import com.intellij.psi.search.IndexPatternOccurrence
+import com.intellij.psi.search.searches.IndexPatternSearch
+import com.intellij.util.Processor
+import nl.hannahsten.texifyidea.file.LatexFile
+import nl.hannahsten.texifyidea.util.files.commandsInFile
+import nl.hannahsten.texifyidea.util.magic.CommandMagic
+import nl.hannahsten.texifyidea.util.matches
+
+/**
+ * Provides the "to do" item in the toolwindow and highlighting in the editor.
```

Comment:
This doesn't do any highlighting, right? At least I didn't observe any change in highlighting after disabling this